### PR TITLE
enh(Resource-status/Graph): optimize lisibility of the Graphs

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This enhances the display and the user experience of the performance graphs for Resources:
- Displays a single metric by clicking on it (unselects all others)
- If the metric is the only one selected and is clicked, displays all metrics
- Toggles a metric by ctrl + clicking on it
- Enhances the tooltip metric display
- Allows the tooltip to be displayed outside the panel
- Optimize the length of the legend by truncating if it doesn't fit the panel width 

graph-enh

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

**Review by Thebarda — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Comment by jdelpierre** [comment]
I've tried with some graph data, and it's correct. 
image

File: www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     flexDirection: 'column',
   },
-  dateTime: {
+  emphasized: {
```
Comment (jdelpierre) [inline www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx:27]: Why this term ? "emphasized" ?

**Review by kduret — APPROVED** [review]

File: www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx (hunk 1 above)
Comment (kduret) [reply www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx:27]: You can check emphasis here : https://material.io/design/color/text-legibility.html#text-backgrounds
in french, It's like "mettre en valeur"

Comment (jdelpierre) [reply www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx:27]: Yes ! ok thanks :)

**Review by jdelpierre — APPROVED** [review]

File: www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx (hunk 1 above)
Comment (bdauria) [reply www/front_src/src/Resources/Graph/Performance/Graph/MetricsTooltip.tsx:27]: it means that the item is presented with more importance

**Pull request merged**